### DISCUSSION TITLE
how to plot performance curves of  trained models？

### DISCUSSION THREAD

**Discussion by rafealzheng** [autogluon/autogluon#2237] [Q&A]
hi，i'm a new to autogluon. When I have trained model, fit_summary() only show results. I want to plot some curves during fitting. So which file in the savepath is the logfile, and how can I use tensorboard or something other to plot ?**

**Comment by ArijitSinghEDA** [comment]
I come here with the same question. With no support for callbacks, how this will be achieved?

**Reply by rafealzheng** [reply]
Finally I ploted curves in using matplotlib with the stored data
